### PR TITLE
Keep track of live registers during bytecode meta-compilation

### PR DESCRIPTION
**Pull request by guillep**
Keep track of live registers during dynamic allocation in bytecode compilation.

Requires changes in the latest VM to detect the failure to allocate registers.

Part of #55

### CODE REVIEW

**Comment by guillep** [comment]
If you wait for me, next week I can add tests.

In any case, the CI will not pass without my changes in the VM :)

File: Druid/DRCogitStackToRegisterMappingGenerator.class.st [hunk 1]
```diff
@@ -12,24 +12,44 @@ Class {
 DRCogitStackToRegisterMappingGenerator >> allocateVariable: aVariableName [
```
Comment (PalumboN) [inline Druid/DRCogitStackToRegisterMappingGenerator.class.st:12]: What is `aVariableName`? I think it is `aDRResult`. In that case, we could rename the variable... and the method.

**Review by PalumboN — DISMISSED** [review]
Cooollll! 

Don't forget to make the PR with the changes in the VM.

And yes, tests 🚥

**Review by PalumboN — APPROVED** [review]

**Comment by guillep** [comment]
Tests are green!

**Pull request merged**